guides/source/layout.html.erb

```diff
@@ -14,6 +14,15 @@
   <script src="javascripts/turbolinks.js" data-turbolinks-track="reload"></script>
   <script src="javascripts/guides.js" data-turbolinks-track="reload"></script>
   <script src="javascripts/responsive-tables.js" data-turbolinks-track="reload"></script>
+  <meta property="og:title" content="<%= yield(:page_title) || 'Ruby on Rails Guides' %>" />
+  <% if content_for?(:description) %>
```

Comment:
I'm not sure if I understand this correctly, but I would assume that `@description` would always be set since you always call `generate_description`?

Reply:
Not always. For example, index.html doesn't have `@header`, so I added `ontent_for?(:description)` check.